packages/devtools_app/lib/src/debugger/evaluate.dart

```diff
@@ -47,6 +47,7 @@ class _ExpressionEvalFieldState extends State<ExpressionEvalField>
         onTap: _onSelection,
         bottom: false,
         maxWidth: false,
+        autoCompleteMatchTileHeight: 50.0,
```

Comment:
This seems way too high for eval. I can imagine a higher value for file search as you are showing 2 lines of text for each entry.
Please include before and after screenshots to clarify.
You'll also need to scale this value by the font size so it still works well with larger or smaller fonts.

Reply:
Dug into this a bit more. The formatting gets very strange for anything less than 48px in height (see photos below),  which is what Flutter is defining as the default list tile height for one line of text & dense spacing: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/list_tile.dart#L1379-L1380

I'm guessing this is because the Material specs says that clickable elements should be at least 48px for accessibility. So if we want it to be any narrower, we would probably have to use a different widget (maybe row?) 

I've changed the height to be whatever is larger of 48px or the text height. That way we don't need the `autoCompleteMatchTileHeight` parameter at all. 

How do I choose a larger font size to verify this scales nicely?


**48 px (current minimum height):**
![Screen Shot 2021-09-21 at 2 17 26 PM](https://user-images.githubusercontent.com/21270878/134249723-5f58913c-c51b-4737-ba9d-689d4fc029be.png)

**40px:**
![Screen Shot 2021-09-21 at 2 05 22 PM](https://user-images.githubusercontent.com/21270878/134249641-22ad6a6e-d47c-4cef-984a-8068214439c5.png)

**30px:**
![Screen Shot 2021-09-21 at 2 05 02 PM](https://user-images.githubusercontent.com/21270878/134249663-f7118844-c206-4969-a2d6-686796785ae6.png)

**20px:**
![Screen Shot 2021-09-21 at 2 04 42 PM](https://user-images.githubusercontent.com/21270878/134249698-24aa59fe-d3d5-44ee-8ef5-f9095baa4ec0.png)

Reply:
we should write our own trivial widget for this. Even 30px looks like it is adding too much space.

Reply:
Sg, removed ListTile! This is what it looks like now:
![Screen Shot 2021-09-22 at 10 24 49 AM](https://user-images.githubusercontent.com/21270878/134397618-fafec7e9-d9a4-4bff-84ca-938771df2a1a.png)